OrchardCore.Commerce/Recipes/OrchardCore.Commerce.Setup.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "name": "OrchardCore.Commerce.Setup",
+  "displayName": "OrchardCore Commerce",
+  "description": "Main setup recipe for the OrchardCore Commerce platform.",
+  "author": "OrchardCoreCommerce",
+  "website": "https://www.orchardcorecommerce.com",
+  "version": "1.0",
+  "issetuprecipe": true,
+  "categories": ["orchardcorecommerce"],
+  "tags": [ "setup" ],
+
+  "steps": [
```

Comment:
Like I said, execute the OCC-specific recipes:
- https://github.com/OrchardCMS/OrchardCore.Commerce/blob/main/OrchardCore.Commerce/Recipes/Order.recipe.json
- https://github.com/OrchardCMS/OrchardCore.Commerce/blob/main/OrchardCore.Commerce/Recipes/Product.recipe.json